### DISCUSSION TITLE
How can I change my fork of this repo to show links to my own hosted web version?

### DISCUSSION THREAD

**Discussion by CycloKid** [MrMeowzz/Funkin-MrMeowzz#67] [Q&A] [answered]
I can't find an index.html file, so I would have to type out the URL manually when I want to play with the web version

**Comment by MrMeowzz** [comment]
What do you mean exactly?
The index.html file is located in the web folder [here](https://github.com/MrMeowzz/Funkin-MrMeowzz/blob/main/web/index.html) if that's what you're wondering.

**Reply by CycloKid** [reply]
I mean like the links section on the site.
Here:
![image](https://user-images.githubusercontent.com/63879422/168073307-96291cab-cba0-4d51-b650-7a0edc133974.png)
Trying to click these links just leads back to your GitHub site (which is blocked 😢)
All I need is to have these 2 links (web version links) to go to my own site. ([https://cyclokid.github.io/Funkin-MrMeowzz-fork/](url))

**Reply by MrMeowzz** [reply]
You would have to change the link in the README.md file for it to change on the site as well then.

**Reply by CycloKid** [reply]
oh! thanks, I was kinda confused since there wasn't an index.html file, but a readme file instead.

**Comment by CycloKid** [comment, marked as answer]
Ok then, just edit the readme.md file instead
